### PR TITLE
Eint 32 update readme rokt rokt sdk ios

### PR DESCRIPTION
**Pull request by mirannakamura**
### Background ###

Update Readme on this repository re: https://rokt.atlassian.net/browse/EINT-32

### What Has Changed: ###

Added to readme: resident experts, license, requirements, project structure, installation and testing steps

### How Has This Been Tested? ###

Installation and testing steps have been tested on my own machine and confirmed working
Screenshot 2023-01-24 at 11 43 26 am
Screenshot 2023-01-25 at 4 56 00 pm

### Checklist: ###

- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have verified that CI completes successfully.
- [x] All insignificant commits have been squashed.

### CODE REVIEW

**Comment by mirannakamura** [comment]
resquash commits

**Comment by emmanuel-rokt** [comment]
LGTM. The simulators on M1 Macs boot too quickly. Adding that delay ensures the placement appears

File: README.md [hunk 1]
```diff
@@ -1,22 +1,111 @@
...
+
+```swift
+func showWidget() {
+    let attributes = ["email": "j.smith1674613477652@example.com",
+            "firstname": "Jenny",
+            "lastname": "Smith",
+            "mobile": "(555)867-5309",
+            "postcode": "90210",
+            "country": "US",
+            "sandbox": "true"]
```
Comment (danial-rokt) [inline README.md:84]: Sandbox = true would force the SDK to use the sandbox environment and they need to be aware of it. We have this part separately in https://docs.rokt.com/developers/integration-guides/ios/reference/sandbox-integration

File: README.md [hunk 2]
```diff
@@ -1,22 +1,111 @@
...
 ```
 
-For more information please visit [official docs](https://docs.rokt.com/docs/developers/integration-guides/ios/overview)
+### Note on Objective-C integration
+
+The Rokt SDK for iOS is implemented using Swift. If you are using Objective-C, you need to import the bridging header file from the framework into the .h/.m file.
+```objective-c
+#import <Rokt_Widget/Rokt_Widget-Swift.h>
+```
+If you are having trouble installing and are receiving an error saying that the SWIFT_VERSION is not defined, please add a user-defined variable `SWIFT_VERSION`. This variable should be set to `5` for iOS SDK versions 2.0 and above. This variable should be set to `4.2` for iOS SDK version 1.2.1.
```
Comment (danial-rokt) [inline README.md:53]: We don't have SDK version 1.2.1 active anymore. You could remove it entirely.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,111 @@
...
+
+Contact Rokt for the Rokt Account ID associated with your account and enter your unique Rokt Account ID as the `roktTagId`.
+
+### 2. Execute (overlay placement)
+
+Firstly define in `ViewController` e.g.
+
+```swift
+func showWidget() {
+    let attributes = ["email": "j.smith1674613477652@example.com",
```
Comment (danial-rokt) [inline README.md:78]: Better to use another email address. This is a text email.

File: README.md [hunk 4]
```diff
@@ -1,22 +1,111 @@
...
+    }, onEmbeddedSizeChange: { selectedPlacement, widgetHeight in
+        // Optional callback to get selectedPlacement and height required by the placement every time the height of the placement changes
+    })
+}
+```
+
+Replace `viewName` in the above snippet with your configured view name
+
+Finally, call this function when the placement needs to be shown.
+**Important** Function needs to be called either in a delay function (snippet below) or on button click to allow some time for initialization.
```
Comment (danial-rokt) [inline README.md:103]: Need to make sure the partner understands what is the issue. We need to ensure they don't call it in the first view of the application. 
Suggesting to use DispatchQueue in the official documentation could cause some confusion as it is not a proper way.

Comment (emmanuel-rokt) [reply README.md:103]: probably ok as the readme is private, yeah?

Comment (danial-rokt) [reply README.md:103]: No this readme is not private. It is the public repository and the readme would be shipped as part of the Swift package manager.

File: README.md [hunk 5]
```diff
@@ -1,22 +1,111 @@
 # Rokt iOS SDK
 
-The Rokt iOS SDK enables you to integrate Rokt into your native iOS mobile apps to drive more value from—and for—your customers. The SDK is built to be lightweight, secure, and simple to integrate and maintain, resulting in minimal lift for your engineering team.
+The Rokt iOS SDK enables you to integrate Rokt into your native iOS mobile apps to drive more value from, and for, your customers. The SDK is built to be lightweight, secure, and simple to integrate and maintain, resulting in minimal lift for your engineering team.
 
-## Swift Package Manager Integration
+## Resident Experts
 
-The [Swift Package Manager](https://swift.org/package-manager/) is a tool for automating the distribution of Swift code and is integrated into the `swift` compiler.
+- Emmanuel Tugado - emmanuel.tugado@rokt.com
```
Comment (danial-rokt) [inline README.md:7]: I would suggest removing our name as the readme is going to be shipped with the SDK and we don't want to have individual names there.

**Pull request merged**